credmgr

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+function credmgr_on() {
+	if ! [ -e "${HOME}/.aws/credentials.normal" ]; then
```

Comment:
@waucka We use `~/.aws/config` to find a profile to use to _set up_ the bucket, but `aws-sdk-go` uses `~/.aws/credentials` so that's where we put the creds for the secretshare user that'll be doing the S3 uploading